Makefile

```diff
@@ -264,6 +264,16 @@ uninstall: uninstall-xinetd uninstall-launchd uninstall-systemd
 	rmdir -p $(SBINDIR) $(DOCDIR) 2>/dev/null || true
 	@echo
 
+uninstall-inetd:
+	if ( -f "$(INETD)" ]; then \
+		@echo "Please manually delete following line and restart or"
+		@echo "kill -HUP the inetd process."
+		@echo
+		@echo "
```

Comment:
what is this line for?

Reply:
Telltales of a bit of a mess.  I'll correct all of it and push my results.